src/content/en/fundamentals/payments/android-pay.md

```diff
@@ -2,14 +2,17 @@ project_path: /web/fundamentals/_project.yaml
 book_path: /web/fundamentals/_book.yaml
 description: Android Pay enables simple and secure purchases online and eliminates the need for users to remember and manually enter their payment information. Integrate Android Pay to reach millions of Android users, drive higher conversion, and give users a true one-touch checkout experience.
 
-{# wf_updated_on: 2017-04-21 #}
+{# wf_updated_on: 2017-10-27 #}
 {# wf_published_on: 2016-09-07 #}
 
 # Integrating Android Pay into Payment Request {: .page-title }
```

Comment:
Should the title change?

Reply:
This page is unlinked and suggesting to go to a new page under /payments. So no title change is required here.